ProjFS.Mac/PrjFSKext/VnodeCache.cpp

```diff
@@ -367,13 +433,17 @@ KEXT_STATIC bool TryFindVnodeIndex_Locked(
     /* out parameters */
     uintptr_t& vnodeIndex)
 {
+    AtomicFetchAddCacheHealthStat(VnodeCacheHealthStat_TotalCacheLookups, 1ULL);
```

Comment:
Since we're under a shared lock I understand the driver for making this operation atomic.
Where we are under an exclusive lock do we need to be there atomic too?

Reply:
See related discussion in earlier comment, because the stats have atomic types `++` and `+=` will also be atomic.

I went with `atomic_fetch_add` to make it explicit that we expect these to be atomic types.